enhancements/machine-api/cluster-api-integration.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Cluster API Integration
+authors:
+  - "@JoelSpeed"
+  - "@alexander-demichev"
+reviewers:
+  - "@elmiko"
+  - "@Fedosin"
+  - "@lobziik"
+  - "asalkeld"
+  - "hardys"
+approvers:
+  - TBD
+creation-date: 2021-09-16
+last-updated: 2021-09-16
+status: implementable
+---
+
+# Cluster API Integration
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement describes the process of integrating the upstream [Cluster API](https://github.com/kubernetes-sigs/cluster-api)
+project into OpenShift standalone clusters.
+
+## Motivation
+
+We would like to give users the ablility to use Cluster API for machine management, as an alternative and possible future replacement for Machine API.
+
+### Goals
+
+- Run Cluster API controllers for managing infrastructure in a similar way to Machine API
+- Provide forward compatibility between Machine API (MAPI) and Cluster API (CAPI)
+- Ensure feature parity between MAPI and CAPI before migration
+
+### Non-Goals
+
+- Deprecate or remove any existing APIs
+- Stop providing support for Machine API in near future
+
+## Proposal
+
+In the longer term, our goal is to reconverge with the upstream Cluster API project and leverage the community built providers for managing Machines within OpenShift standalone clusters.
```

Comment:
i'm a little hesitant about stating our goal to reconverge with CAPI here. afaik we are still evaluating making that choice. especially given that the following sentences talk about using this as an evaluation.

Reply:
I think you're right, this is an evaluation tool for us, to see if we think this is feasible and assess feedback on running CAPI within OCP. Though I don't think there's necessarily anything wrong with this sentence. The reason we are running this TechPreview is to evaluate whether our longer term goal of reconverging is viable.

Is there a way you would reword this to make it more clear? The next sentence is talking about how this preview allows us to gather feedback on the feasability of the goal